sentry_kube/cli/quickpatch.py

```diff
@@ -2,13 +2,14 @@
 from libsentrykube.git import go_to_main, pull_main, create_branch
 from libsentrykube.quickpatch import apply_patch, get_arguments
 from libsentrykube.kube import render_templates
-from typing import Sequence, Mapping
+from typing import MutableMapping, Sequence, Union
 
 __all__ = ("quickpatch",)
 
 
 @click.command()
 @click.option("--service", "-s", help="Sentry kube service name")
+@click.option("--region", "-rg", help="Sentry region name. For example: us, saas, eu")
```

Comment:
You do not need an argument for this. 

Region is pre-populated with `ctx.obj.customer_name` , Cluster is pre-populated with `ctx.obj.cluster_name`